hasher-matcher-actioner/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -29,6 +29,14 @@ class BankedContentMetadata(t.TypedDict):
     json: t.NotRequired[dict[t.Any, t.Any]]
 
 
+class BankContentResponse(t.TypedDict):
+    id: int
+    bank: str
+    enabled: bool
+    original_media_uri: t.Optional[str]
+    signals: t.Dict[str, str]
```

Comment:
You can denote this key is optional with NotRequired (https://typing.readthedocs.io/en/latest/spec/typeddict.html#required-and-notrequired), which I think is fine since our min version for HMA is high enough (3.11)

```suggestion
    signals: dict[str, str]
```